

```diff
@@ -0,0 +1,370 @@
+
+    Merge pull request #4 from google/KevinFormat
+    
+    Formatting Java files using Google's public formatting library.
+
+[33mcommit bbf9e5fbc38690e1c62bec07235284eb6dc2d0af[m
+Merge: d85d92b 1c18506
+Author: Kevin Workman <kevinaworkman@gmail.com>
+Date:   Sun Feb 18 10:52:19 2018 -0800
+
+    Merge branch 'master' into KevinFormat
+
+[33mcommit 1c18506114e3be3017b0ec2e4d8742c8bbafef5c[m
+Merge: 298143c bec6242
+Author: Kevin Workman <kevinaworkman@gmail.com>
+Date:   Sun Feb 18 10:39:20 2018 -0800
+
+    Merge pull request #3 from google/KevinDevelopment
+    
+    Added MessageStoreTest
+
+[33mcommit d85d92bcbb147b9584f78a23845935d43690e49e[m
+Author: Kevin Workman <kevinaworkman@gmail.com>
+Date:   Sun Feb 18 10:36:07 2018 -0800
+
+    Using the @return tag.
+
+[33mcommit bec6242a892296013875d27fa66f139a00987ce8[m
+Author: Kevin Workman <kevinaworkman@gmail.com>
+Date:   Sun Feb 18 10:17:48 2018 -0800
+
+    Fixing spelling of resultMessage variable.
+
+[33mcommit 2760c0d87a9831c659ce9cc40c69a555cb70a1f1[m
+Author: Kevin Workman <kevinaworkman@gmail.com>
+Date:   Sat Feb 17 12:53:36 2018 -0800
+
+    Formatting Java files using Google's public formatting library.
+
+[33mcommit 1a65b41bd8feae4027ee1ab9892d0d00f4517f27[m
+Author: Kevin Workman <kevinaworkman@gmail.com>
+Date:   Fri Feb 16 11:04:55 2018 -0800
+
+    Added MessageStoreTest
+
+[33mcommit 298143ce3ea2e248761e75d722226c42a16abbf7[m
+Merge: 07ece4e 8cf3ed7
+Author: Kevin Workman <kevinaworkman@gmail.com>
+Date:   Fri Feb 16 09:16:11 2018 -0800
+
+    Merge pull request #2 from google/KevinDevelopment
+    
+    Adding ConversationStoreTest
+
+[33mcommit 8cf3ed732e243ef4d24655e9005e8e9e2af10215[m
+Author: Kevin Workman <kevinaworkman@gmail.com>
+Date:   Mon Feb 12 16:07:04 2018 -0800
+
+    Adding ConversationStoreTest.
+
+[33mcommit f491de5888bab81fc923fb558c403cc048fda0c0[m
+Author: Kevin Workman <kevinaworkman@gmail.com>
+Date:   Mon Feb 12 16:01:53 2018 -0800
+
+    Adding ConversationStoreTest.
+
+[33mcommit 07ece4e95c1ca628356bf0e79b0717f74957f5be[m
+Merge: fd00e34 0a42c0f
+Author: Kevin Workman <kevinaworkman@gmail.com>
+Date:   Tue Feb 6 15:40:01 2018 -0800
+
+    Merge pull request #1 from google/KevinDevelopment
+    
+    Adding UserStoreTest.
+
+[33mcommit 0a42c0f40b2962c4f7061176d46888d73fc06d39[m
+Author: Kevin Workman <kevinaworkman@gmail.com>
+Date:   Tue Feb 6 14:20:28 2018 -0800
+
+    Making userList final.
+
+[33mcommit cf5ac19299ed8582ad4c5baee93e1caa5a1e9b75[m
+Author: Kevin Workman <kevinaworkman@gmail.com>
+Date:   Tue Feb 6 13:43:08 2018 -0800
+
+    Adding UserStoreTest.
+
+[33mcommit fd00e34c90042bdf33ffae8880caace6108a9866[m
+Author: Kevin Workman <kevinaworkman@gmail.com>
+Date:   Tue Feb 6 12:52:16 2018 -0800
+
+    Update README.md
+
+[33mcommit 80fa7059e16763ba1497b327d304a42a7c47138e[m
+Author: Kevin Workman <kevinaworkman@gmail.com>
+Date:   Tue Feb 6 12:47:49 2018 -0800
+
+    Initial commit.
```

Comment:
I'm not sure which file shows this, but I don't think it should be included in the merge since it's a list of commit records?